modin/pandas/dataframe.py

```diff
@@ -634,10 +634,17 @@ def align(
         fill_axis=0,
         broadcast_axis=None,
     ):
-        raise NotImplementedError(
-            "To contribute to Pandas on Ray, please visit "
-            "github.com/modin-project/modin."
-        )
+        return self._default_to_pandas_func(op="align",
```

Comment:
instead of strings, can we use the actual function `pandas.DataFrame.align`?

Same for all of these